camayoc/tests/qcs/api/v1/scanjobs/test_run_scanjobs.py

```diff
@@ -303,3 +311,101 @@ def test_scan_task_results(scan_info):
         num_failed = task['systems_failed']
         num_scanned = task['systems_scanned']
         assert num_scanned == sys_count - num_failed
+
+
+def test_disabled_optional_products_facts():
+    """Test scan jobs from scans with disabled optional products.
+
+    :id: 6f91ea5c-32b9-11e8-b467-0ed5f89f718b
+    :description: Test that scan jobs of scans with disabled products
+        are not gathering the raw_facts defined in the roles of the
+        disabled products.
+    :steps:
+        1) Iterate over the scans and gather the disabled_optional_products
+        2) For any scan that was defined with disabled_optional_products:
+            a) Create a facts_to_ignore list composed of raw_facts for each
+                disabled role
+            b) Iterate through the inspection results fact dictionaries
+            c) Assert that no facts that should be ignored are in the
+                dictionaries
```

Comment:
The tool that consumes these doc strings that turns them into test cases on polarion (a reporting tool) wants there to be another section `:expectedresults:`
So maybe `:expectedresults: No facts are collected for the disabled products`

Reply:
Fixed